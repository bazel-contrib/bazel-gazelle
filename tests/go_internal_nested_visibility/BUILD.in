# gazelle:prefix example.com/m
