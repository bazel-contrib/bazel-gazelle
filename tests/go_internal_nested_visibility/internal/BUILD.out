load("@io_bazel_rules_go//go:def.bzl", "go_library")

go_library(
    name = "internal",
    srcs = ["internal.go"],
    importpath = "example.com/m/internal",
    visibility = ["//:__subpackages__"],
)
